README.md

```diff
@@ -16,6 +16,10 @@ $ docker run --rm -it -v $PWD/images:/freetz/images mtneug/freetz
 $ docker run --rm -it -v $PWD/images:/freetz/images -v $PWD/config:/.config mtneug/freetz
 ```
 
+# Build with patches
+# put your .patch files into $PWD/patches
+$ docker run --rm -it -v $PWD/patches:/patches -v $PWD/images:/freetz/images mtneug/freetz
+
```

Comment:
I assume you wanted to include those lines in the code block? Could you then move the three backticks below the last line, as GitHub would otherwise interpret it as Markdown.